kube_hunter/modules/discovery/kubectl.py

```diff
@@ -32,7 +32,7 @@ def get_kubectl_binary_version(self):
         version = None
         try:
             # kubectl version --client does not make any connection to the cluster/internet whatsoever.
-            version_info = subprocess.check_output("kubectl version --client", stderr=subprocess.STDOUT)
+            version_info = subprocess.check_output(["kubectl", "version", "--client"], stderr=subprocess.STDOUT)
```

Comment:
Not that it would make much difference, but we should prefer using the built-in [shlex](https://docs.python.org/3.9/library/shlex.html) module for this.

```suggestion
            version_info = subprocess.check_output(shlex.split("kubectl version --client"), stderr=subprocess.STDOUT)
```

Reply:
@martbhell You might need to import the library before using it. :)